recipes/wcp/icon/v1/a100/environments.yaml

```diff
@@ -14,10 +14,11 @@ nvidia:
   - eccodes@2.25.0%nvhpc +tools +fortran +aec +openmp jp2k=jasper
   - cmake%gcc
   - cuda@12.3%gcc
-  - hdf5%gcc
+  - openblas%gcc
   - hwloc%gcc
-  - netcdf-c%gcc
-  - netcdf-cxx4%gcc
+  - hdf5+szip+fortran%nvhpc
+  - netcdf-c%nvhpc
+  - netcdf-cxx4%nvhpc
```

Comment:
As a rule, we try to compile all C/C++ packages using gcc (because it is a better C/C++ compiler), and only use nvhpc for fortran packages.

Did you change to `nvhpc` because there was a problem with using `gcc`?

Reply:
yes, concretizer didn't want to use it and recompiled one with nvhpc